webservices/rest.py

```diff
@@ -476,6 +514,17 @@ def api_ui():
     )
 
 
+@app.route('/report-csp-violation/', methods=['POST'])
+def report():
+    """
+    Log Content Security Policy (CSP) violations from the browser
+    for both API and CMS.
+    Reports come in with tag 'csp-report'
+    """
+    app.logger.info(ujson.loads(str(request.data, 'utf-8')))
+    return util.output_json("CSP violation reported", 200)
```

Comment:
without the `unsafe-inline` tag the swagger docs are not loading on the browser. And the console error prevents the user from further accessing the swagger docs, which looks right.  But on the API,  we are logging this as a successful POST request with status_code 200.  I think we should change the log from INFO to ERROR and pass a different  status_code 405.
More on status codes :https://www.restapitutorial.com/httpstatuscodes.html

Reply:
@pkfec the 200 status code response is for the reporting URL endpoint, so it makes sense to keep it as 200 to indicate the report was properly received. The `error` log level usually indicates an application error, and in this case the application is working as expected, so it makes more sense to me to keep it at `info`.

Reply:
@pkfec I can't get `report-to` to work and it's only supported in Chroms, so I think I'd like to keep `report-uri` for now, since it's still supported.

Reply:
`report-uri` is supported for now on all the browsers `except` IE.

Reply:
@lbeaufort Would it add more value, if we send a slack msg (alerts channel) or send an email to the team  that the CSP violation has occurred than just logging with 200 status code?